designer/server/src/test/scala/pl/touk/nussknacker/ui/process/migrate/StandardRemoteEnvironmentSpec.scala

```diff
@@ -185,56 +253,49 @@ class StandardRemoteEnvironmentSpec
         uri.toString.startsWith(s"$baseUri$relative") && method == m
       }
 
-      object Validation {
-        def unapply(arg: (String, HttpMethod)): Boolean = is(s"/processValidation/${expectedProcessDetails.name}", POST)
-      }
-
-      object UpdateProcess {
-        def unapply(arg: (String, HttpMethod)): Boolean = is(s"/processes/${expectedProcessDetails.name}", PUT)
-      }
-
-      object CheckProcess {
-        def unapply(arg: (String, HttpMethod)): Boolean = is(s"/processes/${expectedProcessDetails.name}", GET)
+      object GetMigrationScenarioDescriptionVersion {
+        def unapply(arg: (String, HttpMethod)): Boolean = is("/migration/scenario/description/version", GET)
       }
 
-      object AddProcess {
-        def unapply(arg: (String, HttpMethod)): Option[Boolean] = {
-          if (is(s"/processes", POST)) {
-            parseBodyToJson(request).hcursor.downField("isFragment").as[Boolean].toOption
-          } else {
-            None
-          }
-        }
+      object Migrate {
+        def unapply(args: (String, HttpMethod)): Boolean = is("/migrate", POST)
       }
       // end helpers
 
       (uri.toString(), method) match {
-        case Validation() =>
-          Marshal(ValidationResult.errors(Map(), List(), List())).to[RequestEntity].map { entity =>
+        case GetMigrationScenarioDescriptionVersion() =>
+          Marshal(ApiVersion(scenarioDescriptionVersion)).to[RequestEntity].map { entity =>
             HttpResponse(OK, entity = entity)
           }
-        case CheckProcess() if remoteProcessList contains expectedProcessDetails.name =>
-          Marshal(expectedProcessDetails).to[RequestEntity].map { entity =>
-            HttpResponse(OK, entity = entity)
-          }
-        case CheckProcess() =>
-          Future.successful(HttpResponse(NotFound))
-        case AddProcess(isFragment) =>
-          remoteProcessList = expectedProcessDetails.name :: remoteProcessList
-          triedToAddProcess = true
-          addedFragment = Some(isFragment)
-
-          Marshal(ProcessTestData.validScenarioDetailsForMigrations).to[RequestEntity].map { entity =>
-            HttpResponse(OK, entity = entity)
+        case Migrate() =>
+          /*
+            FIXME: Currently, MigrateScenarioRequestDtoV1 and MigrateScenarioRequestDtoV2 are "the same" and they serve
```

Comment:
TBH we should consider dropping the MigrateScenarioRequestDtoV2 (as a last phase of the review changes). Or just comment it out and add a proper note. 

It's because V1 and V2 are the same. At the moment we recommend changing the shape of V2 when the first inconsistency between scenario versions takes place. But from the API perspective, it would be a breaking change. So, maybe it's better to focus on the fact that currently we have only one version and make sure (by making use of comment nodes) that in the future, another developer will be able to add the new version.

Reply:
I would change the comment to encourage the user to add `V3`. The reasons is having comments is not so restrictive and might lead to catastrophic bugs.

Reply:
But do we need to have `MigrateScenarioRequestDtoV2` at the moment? If it's only for the example purpose, maybe we can just comment out all the code related to V2 and in all these places we could leave a note that it can be uncommented and adopted. WDYT?

Reply:
Okay, that would be good compromise